### ISSUE TITLE
wrong cup.avgLoad calculation 

### ISSUE THREAD

**Issue by qoomon** [sebhildebrandt/systeminformation#921]
**Describe the bug**
https://github.com/sebhildebrandt/systeminformation/blob/bb006e4ae2ffc21558e7d31622a673e3b12a38f0/lib/cpu.js#L1583-L1584

The calculation seems to be wrong, because `os.loadavg()` `Returns an array containing the 1, 5, and 15 minute load averages.` see https://nodejs.org/api/os.html#osloadavg therefore it does not make sense to divide these average load numbers by the count of cpu cores `os.loadavg().map(function (x) { return x / util.cores(); });`

**Comment by sebhildebrandt** [comment]
@qoomon, the load average is a Unix-specific concept. A computer with one CPU core that has a single process running at 100% all of the time would have a load average of 1.0. Similarly, on a computer with four CPU cores, a load average of 4.0 indicates the CPUs are being utilized 100%. So therefore it makes sense to divide it by the number of cores.

**Comment by qoomon** [comment]
@sebhildebrandt thanks for the response, however your idea of load is wrong see https://search.app/3iU7294tQvKL5w9Y8

**Comment by sebhildebrandt** [comment]
Thank you @qoomon for your response and the link to the article (which I am anyway aware of). To give you a little background, I had several discussions about that in the past using the way Unix is providing it - where there are ver good reasons to do so - or to change it to rebase the avg load to 1 for 100%. The reason for doing that is, that is it much easier to understand. For those who would like to habe the unix style, they can rely on `os.loadavg()`. In this library I decided to go the other way. Hope this makes sense.

**Comment by qoomon** [comment]
I understand and I think its good to have something in percentage as well like `avgUtilization()`. However it is confusing that 
`os.loadavg()` and `cup.avgLoad()` are named almost the same and has different meanings. Thanks for your response.

**Comment by sebhildebrandt** [comment]
@qoomon good point!! In V6 I anyway have the chance to change the naming (which would be a breaking change). Will have an eye on it. Is it ok to close this issue for now?

**Comment by qoomon** [comment]
that's totally fine
